indexer/bag_indexer/src/main/scala/uk/ac/wellcome/platform/archive/indexer/bags/BagsIndexConfig.scala

```diff
@@ -2,41 +2,49 @@ package uk.ac.wellcome.platform.archive.indexer.bags
 
 import com.sksamuel.elastic4s.ElasticDsl._
 import com.sksamuel.elastic4s.requests.mappings.FieldDefinition
+
 import uk.ac.wellcome.platform.archive.indexer.elasticsearch.IndexConfig
 
 object BagsIndexConfig extends IndexConfig {
-  private val suffixTallyFields: Seq[FieldDefinition] =
-    Seq(
-      keywordField("suffix"),
-      intField("count")
-    )
 
   private val fileFields: Seq[FieldDefinition] =
     Seq(
       keywordField("path"),
       keywordField("name"),
-      keywordField("name"),
+      keywordField("suffix"),
       longField("size"),
       keywordField("checksum"),
       keywordField("type")
     )
 
-  private val payloadStatsFields: Seq[FieldDefinition] =
-    Seq(
-      objectField("fileSuffixTally").fields(suffixTallyFields),
-      intField("fileCount"),
-      longField("fileSize")
-    )
+  private val infoFields: Seq[FieldDefinition] = Seq(
+    keywordField("externalIdentifier"),
+    keywordField("payloadOxum"),
```

Comment:
It suddenly strikes me that Payload-Oxum contains both file count and file size as a single value, albeit a little harder to parse. Maybe we don't need that value again?

(Possibly we store a structured representation of the Payload-Oxum?)